.github/workflows/_jobs_sanctuary.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       #
 
       - name: "Clone repository"
-        uses: "actions/checkout@v3.4.0"
+        uses: "actions/checkout@v4.1.1"
```

Comment:
Using old versions as it was rebased incorrectly (by me) on top of #759 .. Will fix ..